app.py

```diff
@@ -273,14 +281,37 @@ def index(category=[]):
         )
 
 
+@app.route('/teams/<slug>/')
+def team_name(slug):
+    team = _get_team_details(slug)
+
+    if team:
+        api_url = ''.join([INSIGHTS_URL, '/tags?slug=', team['slug']])
+        response = _get_from_cache(api_url)
+
+        response_json = json.loads(response.text)
```

Comment:
what if `response_json` returns an empty array?

Reply:
@tbille I think that's fine. An empty array would be falsey in the template and can be handled there.

Reply:
nope I just tried it sends an error because it doesn't know the variables `post` and `tags`